styles.js

```diff
@@ -1,13 +1,89 @@
+import { css } from "styled-components";
 import { createGlobalStyle } from "styled-components";
 
-export default createGlobalStyle`
-  *,
-  *::before,
-  *::after {
+const globalStyles = css`
```

Comment:
I wonder whether it is necessary to import the css here? I now read that the advantage of doing the css can be to make the code more readable, so probably this is what you are going for

Reply:
yes i have to that prettier can format it on save

Reply:
Sometimes you need to use the CSS method from styled-compoents.
In this case, it's required for the highlighting and the auto-formatting with prettier on save.
You can try it yourself in your code.
Without the CSS method, it's just a big string. Without highlighting and autocomplete.